dashboard/src/app/(authenticated)/(diagram)/components/Modals/TaskRun.tsx

```diff
@@ -56,11 +59,21 @@ export const TaskRun: FC<Modal> = ({ data }) => {
                   <div className="">{attempt.endTime}</div>
                 </div>
               </div>
-              {attempt.output && (
-                <div className="mt-2 flex flex-col rounded bg-zinc-500 p-1 text-white">
-                  <h3 className="font-bold">Output</h3>
-                  <pre className="overflow-x-auto">{getVariableValue(attempt.output)}</pre>
+
+              {attempt.error || attempt.exception ? (
```

Comment:
No need to do a ternary here since they are guaranteed to be `oneOf`.

I would prefer the form `attempt.error && ...`

Reply:
Yea didn't know it was oneOf, just wanted to account for it in case it wasn't. I'll update later